client/ayon_houdini/plugins/publish/collect_output_node.py

```diff
@@ -67,8 +67,8 @@ def process(self, instance):
             out_node = node.parm("startnode").evalAsNode()
 
         else:
-            raise KnownPublishError(
-                "ROP node type '{}' is not supported.".format(node_type)
+            raise PublishError(
```

Comment:
I would still prefer `KnownPublishError` instead of `PublishError` if `KnownPublishError` is not gonna be deprecated.

Reply:
Why would you prefer it? :)

Reply:
> Why would you prefer it? :)

Both exceptions are more likely to describe the publish error (to me it sounds like it happens in the extractor), `KnownPublishError` sounds better as the error is `known` before publishing. (It is just me maybe)

Reply:
To me it is more like: as soon as we raise a dedicated error for something - it is technically known? :D

Reply:
[It's also deprecated](https://github.com/ynput/ayon-core/blob/3da898b3440b5fd9ba75887205e033cc834685a2/client/ayon_core/pipeline/publish/publish_plugins.py#L42).

Reply:
Cool let's kill it 🔪 🔫